Translated/Dialog/Dachua/trial_apc_7904001.csv

```diff
@@ -0,0 +1,54 @@
+Begin#0,"""/ci1 1 Oh, well hello there. Think you could you lend me a hand here?"""
+Success#0,"""/ci2 1 ...Thanks for the help. Good luck out there."""
+Failure#0,"""/ci4 3 Things are looking bad! Sorry, I have to retreat."""
+Dead#0,"""/ci5 3 No... Not like this..."""
+Begin_v2#0,"""/ci1 1 Oh, well hello there. Think you could you lend me a hand here?"""
+Success_v2#0,"""/ci2 1 ...Thanks for the help. Good luck out there."""
+Failure_v2#0,"""/ci4 3 Things are looking bad! Sorry, I have to retreat."""
+Dead_v2#0,"""/ci5 3 No... Not like this..."""
+Begin_7002001#0,"""/ci1 2 Uh-oh. This isn't what I expected..."""
+Success_7002001#0,"""/ci2 1 Phew... Looks like we made it."""
+Failure_7002001#0,"""/ci4 3 Ngh... I think I might have broke something... I'm just gonna make my way back to the City, okay?"""
```

Comment:
"broken" something?

Reply:
that line is aina speaking so i thought it fit her better. i can change it if you want

Reply:
Yeah it's probably fine.